Enable "Delete cloud object store container"

pyon-yx opened the pull request:
I tried to enable the button "Delete Cloud Object Store Container".

agrare commented:
@miq-bot cross-repo-tests https://github.com/ManageIQ/manageiq/pull/21489, https://github.com/ManageIQ/manageiq-providers-openstack/pull/741

pyon-yx commented:
```
def delete_cloud_object_store_containers
    assert_privileges("cloud_tenant_delete")
    containers = find_records_with_rbac(CloudObjectStoreContainer, checked_or_params)

    unless containers.empty?
      process_cloud_object_store_container(containers, "destroy")
    end
  end

  def process_cloud_object_store_container(containers, operation)
    return if containers.empty?

    if operation == "destroy"
      containers.each do |container|
        audit = {
          :event        => "cloud_object_store_container_record_delete_initiated",
          :message      => "[#{container.key}] Record delete initiated",
          :target_id    => container.id,
          :target_class => "CloudObjectStoreContainer",
          :userid       => session[:userid]
        }
        AuditEvent.success(audit)
        container.cloud_object_store_container_delete_queue(session[:userid])
      end
      add_flash(n_("Delete initiated for %{number} Cloud Object Store Container.",
                   "Delete initiated for %{number} Cloud Object Store Containers.",
                   containers.length) % {:number => containers.length})
    end
  end
```
I just added these method in cloud_object_store_container_controller.rb.
and added these lines under `def button` method in the same file.
```
when 'cloud_object_store_container_delete'
      delete_cloud_object_store_containers
```
What screenshot do you want me to share?

kavyanekkalapu commented:
@pyon-yx We have testcases to test functionality in each controller file. please check You can check travis build https://app.travis-ci.com/github/ManageIQ/manageiq-ui-classic/jobs/542178928. test cases are failing in this file https://github.com/ManageIQ/manageiq-ui-classic/blob/master/spec/controllers/cloud_object_store_container_spec.rb.

<img width="945" alt="Screen Shot 2021-10-08 at 2 54 56 PM" src="https://user-images.githubusercontent.com/37085529/136611488-b385f393-3531-4c6b-9acc-137e32060903.png">

you can also check this documentation https://github.com/ManageIQ/guides/blob/master/developer_setup/running_test_suites.md to know more about manageiq test cases.  Once you are trying to fix test cases, if you want to run them locally, you can run `bundle exec rspec /spec/controllers/cloud_object_store_container_spec.rb` to verify in your local.
